### ISSUE TITLE
amp-image-lightbox: optional attribute to change text for screen-reader close button.

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#9868]
**Reserved For write/speak/code event**

### Description
In [amp-image-lightbox](https://ampbyexample.com/components/amp-image-lightbox/), users can tap somewhere outside of the image or swipe up/down to close the lightbox, however this is not accessible for blind users who rely on screen-readers to consume the Web. Therefore an alternative approach has been implemented that adds a visually hidden button to `amp-image-lightbox` which is discoverable by screen-readers to close the lightbox.

### The Problem
Currently the `text` for this button is `Close the lightbox` and is not customizable so it can not be changed or translated.

We need to expose an optional attribute (`data-close-button-aria-label`) on `amp-image-lightbox` that, if set, will replace the `text` to the provided value.

### The Fix
- Go to [amp-image-lightbox.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-image-lightbox/0.1/amp-image-lightbox.js#L751) around line 751. Note that the text for the `screenReaderCloseButton` is hard-coded to `Close the lightbox`
- Change the line above so the text becomes the value of the `data-close-button-aria-label` attribute if it is provided, it should still default back to `Close the lightbox` if `data-close-button-aria-label` is not provided.
- Update the [amp-image-lightbox.md](https://github.com/ampproject/amphtml/blob/master/extensions/amp-image-lightbox/amp-image-lightbox.md#attributes) documentation file to include the new optional `data-close-button-aria-label` attribute.

### Step by step
- [x] Claim this issue by adding a comment below.  Please only claim this bug if you plan on starting work in the next day or so.
- [ ] If you aren't too familiar with Git/GitHub, see the [Getting Started End-to-End Guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md) for [an intro to Git & GitHub,](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#intro-to-git-and-github) and [how to get a copy of the code](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#get-a-copy-of-the-amphtml-code).  You can also refer to the [Quick Start Guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-quick.md) for the necessary setup steps with less explanation than the End-to-End guide.
- [ ] Follow the instructions for [building AMP](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#building-amp-and-starting-a-local-server).
- [ ] [Create a Git branch](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#create-a-git-branch) for making your changes.
- [ ] Do **The fix** as outlined above.
- [ ] [Commit your changes](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#edit-files-and-commit-them) frequently.
- [ ] [Push your changes to GitHub](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#push-your-changes-to-your-github-fork).
- [ ] [Create a Pull Request](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#send-a-pull-request-ie-request-a-code-review) and request a code review from me (@aghassemi).  Mention `Fixes #9868` in the description.
- [ ] [Respond to your reviewer's comments](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#respond-to-pull-request-comments) (if any).

Once approved, your changes will be merged.  **⚡⚡⚡Congrats on making your first contribution to the AMP Project!⚡⚡⚡**  You'll be able to see it [live across the web soon](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md)!

Thanks, and we hope to see more contributions from you soon.

### Questions?

If you have questions ask in this issue or on your Pull Request (if you've created one) or see the [How to get help](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#how-to-get-help) section of the Getting Started guide.

**Comment by kellyk** [comment]
I'll take this one!
